spec/Layers/FeatureLayer/FeatureLayerSpec.js

```diff
@@ -230,6 +258,8 @@ describe('L.esri.Layers.FeatureLayer', function () {
 
     layer.createLayers(multiPolygon);
 
+    expect(layer.getFeature(1).getLayers()[0].options.color).to.equal('black');
```

Comment:
@jgravois this is only going to check that `setStyle`/`resetStyle` works on multipolygons. Could you add a specific test for `setStyle`/`resetStyle` when using `L.circleMarker`?